apps/rig_auth/lib/rig_auth/jwt/utils.ex

```diff
@@ -12,19 +12,27 @@ defmodule RigAuth.Jwt.Utils do
   @type claims :: %{required(String.t()) => String.t()}
 
   @spec valid?(String.t()) :: boolean
-  def valid?(jwt) do
+  def valid?("Bearer " <> jwt) do
     jwt
     |> validate
     |> get_error == nil
   end
 
+  def valid?(invalid_jwt) do
+    %{error: "JWT=#{invalid_jwt} is missing token type. Required format is: \"Bearer token\""}
```

Comment:
I was thinking more about something like `invalid_access_token`, based on the spec.